lib/slack/post.rb

```diff
@@ -33,7 +33,7 @@ def self.post_with_attachments(message, attachments, chan = nil, opts = {})
 
 			http = Net::HTTP.new(uri.host, uri.port, config[:proxy_host], config[:proxy_port])
 			http.use_ssl = true
-			http.ssl_version = :TLSv1
+			http.ssl_version = :TLSv1_2
```

Comment:
is there a reason this doesn't use 1.3?

Reply:
Ask and you shall receive: https://github.com/CozyCo/slack-post/pull/22

Reply:
lol thanks